control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -672,6 +672,7 @@ func (r *Reconciler) getChannelContractResource(ctx context.Context, topic strin
 		Ingress: &contract.Ingress{
 			Host:                       receiver.Host(channel.GetNamespace(), channel.GetName()),
 			EnableAutoCreateEventTypes: feature.FromContext(ctx).IsEnabled(feature.EvenTypeAutoCreate),
+			Path:                       receiver.Path(channel.GetNamespace(), resources.MakeChannelServiceName(channel.GetName())),
```

Comment:
@Leo6Leo I'm curious: why do we need this change?

Reply:
The `Path` is not specified in the dataplane contract. Consequently, when a probing request is transmitted, the receiver at the data-plane is unable to correctly navigate and locate the required resource. @Cali0707